docs/content/how-to/automated-machine-management/configure-machines.md

```diff
@@ -0,0 +1,97 @@
+# Configuring Machines in HyperShift
+
+In standalone OpenShift, machine configuration is managed via resources like MachineConfig, KubeletConfig, and ContainerRuntimeConfig inside the cluster, and then applied to a set of machines via a MachineConfigPool. In HyperShift clusters, machine configuration is not managed from within a hosted cluster, but rather via the NodePool API outside the cluster. 
+
+The NodePool field `.spec.config` can be populated with a list of references to configmaps that contain MachineConfig, KubeletConfig, or ContainerRuntimeConfig manifests. This configuration is applied to all machines belonging to that NodePool. In other words, in HyperShift, the NodePool replaces the MachineConfigPool, MachineConfig, KubeletConfig, and ContainerRuntimeConfig resources in standalone OpenShift.
```

Comment:
> In other words, in HyperShift, the NodePool replaces the MachineConfigPool, MachineConfig, KubeletConfig, and ContainerRuntimeConfig resources in standalone OpenShift.

Not sure if this adds much and is kinda confusing.  Maybe "In other words, in HyperShift, the NodePool replaces the MachineConfigPool, and MachineConfigs, KubeletConfigs, and ContainerRuntimeConfigs are encapsulated in configmaps and referenced by the NodePool config field."

But the previous sentence kinda already said this so :man_shrugging:

Reply:
ack, yup, it's kind of redundant. Removed.